src/support/bytemuck.rs

```diff
@@ -0,0 +1,82 @@
+//! Support for the [`bytemuck`](https://crates.io/crates/bytemuck) crate.
+#![cfg(feature = "bytemuck")]
+#![cfg_attr(docsrs, doc(cfg(feature = "bytemuck")))]
+
+use bytemuck::{Pod, Zeroable};
+use ruint::Uint;
+
+// Implement Zeroable for all `Uint` types.
+unsafe impl<const BITS: usize, const LIMBS: usize> Zeroable for Uint<{ BITS }, { LIMBS }> {}
+
+// Implement the `Pod` trait for `Uint` types with a size that is a multiple of
+// 64, up to 1024. Note that implementors must have a size that is divisible by
+// 64, and using `Uint` sizes not divisible by 64 would violate Pod's
+// guarantees potentially leading to undefined behavior.
+macro_rules! impl_pod {
+    ($(($bits:expr, $limbs:expr)),+ $(,)?) => {
```

Comment:
nit: these are really `literal`, as non-literal `expr` are invalid in this context. no change needed